src/components/pages/ProfilePage/index.jsx

```diff
@@ -0,0 +1,91 @@
+//#region > Imports
+//> React
+// Contains all the functionality necessary to define React components
+import React from "react";
+// DOM bindings for React Router
+import { withRouter } from "react-router-dom";
+
+//> Components
+import { SoftwareEngineer } from "../../organisms/profiles";
+//> CSS
+import "./profile.scss";
+//#endregion
+
+//#region > Components
+class ProfilePage extends React.Component {
```

Comment:
Add description: `This component adds the Profile page and distinguishes between software and media technology accounts and loads the components accordingly.`